### ISSUE TITLE
about scipy version

### ISSUE THREAD

**Issue by TnoobT** [vye16/slahmr#21]
hi, bro. very nice work！I want to know what version of scipy you have installed. I installed scipy=1.11.0 and reported an error
![image](https://github.com/vye16/slahmr/assets/44052395/dc251bf0-7ac9-4870-ab54-0e393ab211d6)

**Comment by geopavlakos** [comment]
We provide [this file](https://github.com/vye16/slahmr/blob/main/env_build.yaml) with the full details regarding the versions of the dependencies we use for our environment. The version for scipy is 1.9.1.

**Comment by TnoobT** [comment]
Thank you very much for your reply. I have another question. I tested my own video and when processing frame 7, the code got stuck. Then I debugged and found that the value in the graph was nan, which entered a dead loop. Do you have a way to handle this situation

![image](https://github.com/vye16/slahmr/assets/44052395/3ae9f64c-53a1-4f78-a3b9-f5ad491a480f)

**Comment by geopavlakos** [comment]
Could you share the video?

**Comment by TnoobT** [comment]
I also tested 022691_ Mpii_ Test will also fall into a dead cycle. Is it something wrong with my environment
![image](https://github.com/vye16/slahmr/assets/44052395/7708bf6e-c812-41f8-ad38-0319b0bb6906)
![image](https://github.com/vye16/slahmr/assets/44052395/e940cdd8-e179-4e4d-860a-d0ba5090da49)
![image](https://github.com/vye16/slahmr/assets/44052395/64b08df4-03ac-4733-b790-3a8974bb6df4)
![image](https://github.com/vye16/slahmr/assets/44052395/7ea3dcca-7ee8-4891-b067-40d7575e4d0e)

**Comment by geopavlakos** [comment]
Yes, that video should run without a problem, so it's probably the environment. You can take a look at the [colab](https://colab.research.google.com/drive/1IFvek5DSgKb80vtSvXAXh1xmBFMJuxeL?usp=sharing) to identify differences compared to the default environment and the default installation procedure.

**Comment by TnoobT** [comment]
Thank you for your answer. I have another silly question. I saw in your paper that the SMPL parameters were predicted using humor networks. Have you done any network or loss optimization for the estimation of SMPL parameters

**Comment by geopavlakos** [comment]
Yes, we solve an optimization problem to estimate the SMPL parameters for each person. The HuMoR network is responsible for one of the objectives of this optimization, the motion prior part.

**Comment by TnoobT** [comment]
I ran the code using Colab, but this issue also occurred. Perhaps you need to modify Colab.
![image](https://github.com/vye16/slahmr/assets/44052395/c3299962-ffc9-497f-9907-42f5124855fc)

I made the modifications and now it can work normally
![image](https://github.com/vye16/slahmr/assets/44052395/4c3d6831-9f7c-4c20-8b11-a91cfa05df7b)

**Comment by geopavlakos** [comment]
Thanks for pointing that out! I have updated the colab environment.

**Comment by TnoobT** [comment]
hi, I have reinstalled the environment several times but still fell into a dead cycle. I would like to ask if the version of CUDA on your computer is 11.7, and my version is 11.1. I would like to upgrade my CUDA version

**Comment by geopavlakos** [comment]
We have indeed tested the demo using CUDA version 11.7, but I cannot be entirely sure if the CUDA version is responsible for the issue you are facing.

**Comment by TnoobT** [comment]
ok, thank you. i try it

**Comment by TnoobT** [comment]
Damn it, I know what the reason is. On my own computer, CUDA is 11.1, but in install.sh, CUDA 11.7 corresponds to Torch1.13. I installed Torch1.13, but the program did not report any errors! Then I installed the torch corresponding to cuda11.1 (the torch-scatter also needs to correspond, otherwise the subsequent DROID-SLAM will report an error), the code ran successfully, and finally solved the problem
